package.json

```diff
@@ -49,10 +50,13 @@
 		"read-pkg-up": "^6.0.0"
 	},
 	"devDependencies": {
+		"@types/got": "^9.6.12",
```

Comment:
Got is written by TypeScript. Declarations are [already inside](https://github.com/sindresorhus/got/blob/f896aa52abc41fe40d4942da94a0408477358f14/package.json#L96).